routes/errortracker.js

```diff
@@ -0,0 +1,257 @@
+/**
+ * Created by rigwanzojunior on 6/5/17.
+ *objects used
+ *Objects are identical to the objects used in previous errortracker.go
+ * type errorRequestMeta {
+	HTTPReferrer  string `json:"http_referrer,omitempty"`
+	HTTPUserAgent string `json:"http_user_agent,omitempty"`
+    }
+ type errorRequest {
+	URL    string            `json:"url,omitempty"`
+	Method string            `json:"method,omitempty"`
+	Meta   *errorRequestMeta `json:"meta,omitempty"`
+ }
+ type errorEvent {
+	Application string `json:"application,omitempty"`
+	AppID       string `json:"app_id,omitempty"`
+	Environment string `json:"environment,omitempty"`
+	Version     string `json:"version,omitempty"`
+
+	Message   string `json:"message,omitempty"`
+	Exception string `json:"exception,omitempty"`
+
+	Request *errorRequest `json:"request,omitempty"`
+
+	Filename  string `json:"filename,omitempty"`
+	Line      int32  `json:"line,omitempty"`
+	Classname string `json:"classname,omitempty"`
+	Function  string `json:"function,omitempty"`
+	Severity  string `json:"severity,omitempty"`
+ }
+ * Handle error requests from clients and log them.
+ */
+//AppEngine project ID
+const projectId ='';
```

Comment:
There's frequent spacing issues in these files. To quickly see all issues, install `eslint` into the repo and run with the `extends: google` setting.